modules_basic/reply.js

```diff
@@ -0,0 +1,9 @@
+var h = require('../h')
+
+exports.gives = 'message_action'
+
+exports.create = function () {
```

Comment:
oh, i thought i saw other modules that didn't need anything not receiving api. i'm happy either way.